engine/runtime/src/main/java/org/enso/interpreter/runtime/data/EnsoFile.java

```diff
@@ -523,7 +523,7 @@ public boolean isAbsolute() {
     return this.truffleFile.isAbsolute();
   }
 
-  @Builtin.Method
+  @Builtin.Method(name = "is_directory_builtin")
```

Comment:
Shouldn't `is_directory_builtin` definition appear somewhere in the Enso source code? I don't see it. Thus I cannot change if it is `private`.

Reply:
I have created private module `File_Builtins` inspired by `Number_Builtins` and moved the `File.*_builtin` methods there.